js/data.js

```diff
@@ -6,3 +6,8 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
```

Comment:
You should also add some code here to check if there is any data in localStorage when the page loads, and if there is, replace `data` with that parsed value